src/routes/CreateRoomPage.tsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
+
+function CreateRoomPage() {
+  const handleCreateRoomClick = () => {
```

Comment:
[NIT] is there no backend API call implemented rn cuz it is not set up? If so, please add a comment to document that is the case.